spec/operators/partition-spec.ts

```diff
@@ -255,11 +255,6 @@ describe('Observable.prototype.partition', () => {
     expectSubscriptions(e1.subscriptions).toBe([e1subs, e1subs]);
   });
 
-  it('should throw without predicate', () => {
-    const e1 = hot('--a-b---a------d----');
-    expect(e1.partition).to.throw();
```

Comment:
shouldn't we just correct it into `expect(() => e1.partition.subscribe)).to.throw?`

Reply:
I'm not sure it should throw... it doesn't throw now and none of our other methods throw... for example, if you don't provide a predicate to `filter`.

Perhaps they *should* throw... but it seems unrelated.